pointer-masking-proposal.adoc

```diff
@@ -176,6 +187,9 @@ _**xMTE**_ register fulfills two-fold function:
 1. Can only be programmable by more privileged mode (unless PM.Current bit is enabled) +
 2. Performs status register function for the current privilege mode +
 
+_XS_ bits from **_MMTE_** register are only accessible in M, S and VS-mode. _XS_ field encodes the status of Pointer Masking extension in U and VU-mode. These bits can be checked by a context switch routine to quickly determine whether a state save or restore is required. XS field is automatically updated by the hardware when PM CSR registers are modified. Privileged code can directly modified XS bits to restore previousle save XS status (e.g. by the context restore routine).
```

Comment:
Looking at the Privileged ISA Specification, should changes to the pointer masking CSRs also affect the global XS bits in the mstatus register? If so, it might be worth mentioning this here.

Reply:
Fixed